### DISCUSSION TITLE
Документация Радио86-РК

### DISCUSSION THREAD

**Discussion by mxlgv** [begoon/rk86-js#7] [Q&A]
Здравствуйте! Для "Микроши" я нашёл скан книги, которая шла с ЭВМ в комплекте (инфа там более чем исчерпывающая), а вот для Радио-86РК, почему то подобное найти не могу... Может вы знаете где достать?

**Comment by begoon** [comment]
Много информации про РК есть тут http://www.danbigras.ru/RK86/Index.html.

**Comment by begoon** [comment]
Есть еще вот эта книга [Зеленко Г., Панов С., Попов С, Домашний компьютер](http://myadel-gimnaz.by/sites/default/files/Documents/Instruction/Comp/rk86_book.zip).

**Comment by mxlgv** [comment]
О спасибо! Буду разбираться
